src/client/api.js

```diff
@@ -1,7 +1,7 @@
 const location = window.location;
 
 let API_ROOT;
-if (import.meta.env.prod) {
```

Comment:
It has been like this since we start using vite (v2).

I'm not sure when this stopped working, vite.dev only has migration guide up to v4 -> v5.